roadmap.md

```diff
@@ -23,26 +23,40 @@ implement the spec.
 1. Create and deploy a website that features a simple overview, email list and directs visitors to Github.
 1. Store all website assets in the CloudEvents repository, under the governance
 of the working group.
-
-*0.2*
-
 1. Have at least 2 implementations of the specification that can demonstrate interoperability.
 1. Include a specification for mapping the CloudEvents specification to HTTP.
 1. Include a specification for mapping the CloudEvents specification to JSON.
 1. Changes to the spec to facilitate adoption.
-1. Draft documentation and developer guide.
 1. Publicize at conferences ([CloudNativeCon Europe](https://events.linuxfoundation.org/events/kubecon-cloudnativecon-north-america-2018/)).
-
-*0.3*
-
 1. Interoperability demo.
     1. At least one open source implementation of sending and receiving events.
```

Comment:
suggest addling link, like:

At least one open source implementation of sending and receiving events, see [community open source](https://github.com/cloudevents/spec/blob/master/community/open-source.md)

Reply:
Done